assets/blueprints/blueprint.json

```diff
@@ -0,0 +1,49 @@
+{
+    "landingPage": "/glotpress/",
+    "preferredVersions": {
+      "php": "8.0",
+      "wp": "latest"
+    },
+    "steps": [
+      {
+        "step": "login",
+        "username": "admin",
+        "password": "password"
+      },
+      {
+        "step": "mkdir",
+        "path": "/wordpress/wp-content/languages/plugins"
+      },
+      {
+        "step": "mkdir",
+        "path": "/wordpress/wp-content/languages/themes"
+      },
+      {
+        "step": "setSiteOptions",
+        "options": {
+          "WPLANG": "de_DE",
+          "permalink_structure": "/%year%/%monthnum%/%day%/%postname%/"
+        }
+      },
+      {
+        "step": "writeFile",
+        "path": "/wordpress/wp-content/mu-plugins/gp-sqlite.php",
+        "data": "<?php add_action( 'admin_footer', function() {echo '<input type=\\\"checkbox\\\" name=\\\"gp_enable_inline_translation\\\" checked=\\\"checked\\\" style=\\\"display: none\\\" />';} );add_filter( 'query', function( $query ) {return str_replace( ' BINARY ', ' ', $query );} );add_filter( 'gp_inline_translation_frontend_url', function( $url ) {return 'https://translate.wordpress.org/';} );if ( ! function_exists( 'libxml_clear_errors' ) ) {function libxml_clear_errors() {}}if ( ! function_exists( 'libxml_use_internal_errors' ) ) {function libxml_use_internal_errors() {}}if ( ! function_exists( 'libxml_get_errors' ) ) {function libxml_get_errors() {}}if ( ! function_exists( 'mysql_get_client_info' ) ) {function mysql_get_client_info() {}}if ( ! function_exists( 'mb_str_split' ) ) {function mb_str_split(  $string = '', $length = 1 , $encoding = null ){return str_split( $string, $length );}}if ( ! class_exists( 'DOMDocument' ) ) {class DOMDocument {function loadHTML() {}}}"
```

Comment:
Here also the filter adding the `gp_enable_inline_translation` checkbox is not necessary.